cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -51,6 +51,9 @@
     /dev/pts/[0-9]* rw,
     /dev/tty rw,
 
+    # golang runtime variables
+    /sys/kernel/mm/transparent_hugepage/hpage_pmd_size r,
```

Comment:
I'm unsure why this is needed here. snap-confine only calls galang tools while doing an apparmor profile transition. If anything this should go to snap-update-ns template instead.

Reply:
Oh! Yes, this was meant to be in the snap-update-ns template. Nice catch!

Reply:
Done